packages/sitecore-jss/src/site/graphql-siteinfo-service.ts

```diff
@@ -101,6 +117,18 @@ export type GraphQLSiteInfoResult = {
   };
 };
 
+type GraphQLXmCloudSiteInfoResponse = {
+  site: {
+    siteInfoCollection: GraphQLXmCloudSiteInfoResult[];
+  };
+};
+
+export type GraphQLXmCloudSiteInfoResult = {
```

Comment:
Maybe remove "XmCloud" from these types as well but not a huge deal as they're internal.

Reply:
We are going to remove these in about two months time anyway, I think we can leave them, they serve as a good enough explanation when reading the code